src/gorilla_repl/core.clj

```diff
@@ -99,9 +99,11 @@
     (nrepl/start-and-connect nrepl-requested-port)
     ;; and then the webserver
     (let [s (server/run-server #'app-routes {:port webapp-requested-port :join? false :ip ip})
-          webapp-port (:local-port (meta s))]
+          webapp-port (:local-port (meta s))
+          url (str "http://" ip ":" webapp-port "/worksheet.html")]
       (spit (doto (io/file ".gorilla-port") .deleteOnExit) webapp-port)
-      (println (str "Running at http://" ip ":" webapp-port "/worksheet.html ."))
+      (.. java.awt.Desktop getDesktop (browse (java.net.URI. url))) 
+      (println (str "Running at " url "."))
```

Comment:
I'd simplify this to be:

```clojure
(println "Running at " url ".")
```

I'd even remove the period since it serves no purpose anyway.